server/views/pages/barcode/generate-barcode-image.njk

```diff
@@ -19,6 +19,7 @@
       }) }}
     {% endif %}
 
+    <span id="barcode-created-count" class="govuk-visually-hidden" data-value="{{ barcodeImages | length }}"></span>
```

Comment:
I'm not sure visually hidden is the right class is it? I _think_ that whilst it wont be visible on screen, a screen reader will still read out the element .... so a visually impaired person will hear a random number as part of the reading of the page! 😁 

I think what you want is `display: none`, and I think there is a convenient govuk style you can use. I think its `govuk-!-display-none` (but you might want to check that)